databases/databases.md

```diff
@@ -178,7 +178,7 @@ The next step, once you've had a chance to practice this all in the project, is
 This section contains helpful links to other content. It isn't required, so consider it supplemental for if you need to dive deeper into something.
 
 
-* Odinite Hunter D made his excellent notes into a [Github Book on SQL](http://hgducharme.gitbooks.io/sql-basics/content/) which you should totally check out if you want a decent resource.
+* Odinite Hunter D made his excellent notes into a [Github Book on SQL](https://app.gitbook.com/@hunter-ducharme/s/sql-basics/) which you should totally check out if you want a decent resource.
```

Comment:
Notice the different link here, we should have the same path on both. Notice the `/content`

Reply:
Oh okay. Would you please open the link as they, both link to the same page

Reply:
Ahh, you are correct. Good catch.